spec/models/user_spec.rb

```diff
@@ -3,11 +3,92 @@
 RSpec.describe User, type: :model do
   describe "Associations" do
     it { should have_many(:user_authentications) }
+    it { should have_many(:accesses) }
   end
 
   describe "Validations" do
     it { should validate_presence_of(:full_name) }
     it_behaves_like "a record that validates device timestamps"
+
+    pending { is_expected.to validate_presence_of(:access_level) }
```

Comment:
This is intentionally left pending for when the migration is complete and we add this validation.